Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "stackrankdice"
+version = "0.0.1"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+description = "Dice Wars Game with Bevy"
+keywords = ["bevy", "tabletop", "dice"]
+repository = "https://github.com/quentusrex/stackrankdice"
+
+[dependencies]
+bevy = { version = "0.8.1" }
+bevy_dice = { path = "../bevy_dice" }
```

Comment:
Looks like this is still a local package path instead of a public access package.

Reply:
Thanks for pointing that out. It's guess I might need to add a bevy CI integration to github here.